docs/hints-for-use-with-docker.md

```diff
@@ -0,0 +1,53 @@
+# Hints for using ORT with Docker
+
+## General Hints
+
+### Mount current working directory or directory relative to working directory
+
+```shell
+docker run \
+  -v $PWD/:/project  \ # Mount current working directory into /project to use as input.
+  -v $PWD.ort:/ort \ # Mount folder "<workingdirectory>.ort" to /ort to use as output and config file storage.
```

Comment:
Isn't it a bit odd to use an `.ort` suffix for a directory? Why not a sub-directory in the form "<workingdirectory>/ort"? Or is there a slash *missing* now here, and it should be `$PWD/.ort` to mount the "hidden" `.ort` directory?

Reply:
I actually use it as a suffix, to avoid a directory inside the scanned directory. I'll update it to whatever you see as best practice, this was just personal preference that I used as an example in this instance.